backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5622960_sys_gh12385_Replace_BIC_SwiftCode.sql

```diff
@@ -0,0 +1,18 @@
+
+UPDATE AD_Column
+SET ColumnName ='SwiftCode'
+WHERE AD_Column_ID = 575110
```

Comment:
hmm, this feels awkward because looks like it's done manually.
When changing the column name of an AD_Column i would expect: 
* AD_Element_ID to be changed too
* Name, Description, Help to be changed
* AD_Column_Trl records to be changed
* trls to be propagated

Reply:
@teosarca see https://github.com/metasfresh/me03/issues/10162#issuecomment-1019892033
the ad_element's ColumnName  is 'SwiftCode'
only the AD_Column.ColumnName is BIC. AFAIU someone manually updated it in AD_Column

Reply:
f..k, right!